mkimage

```diff
@@ -78,7 +78,7 @@ repo_url="http://deb.debian.org/debian"
 sec_repo_url="http://security.debian.org/"
 
 echo -e "deb ${repo_url} $DIST main" > "$rootfsDir/etc/apt/sources.list"
-if [ "$DIST" == "bullseye" ]; then
+if [ "$DIST" == "bullseye" ] || [ "$DIST" == "bookworm" ]; then
```

Comment:
It seems with this new version there is a change in the security repo URL.

While in [Bullseye](https://wiki.debian.org/DebianBullseye#FAQ) is `deb http://deb.debian.org/debian-security bookworm-security main`, in [Bookwork](https://wiki.debian.org/DebianBookworm#FAQ) is `http://deb.debian.org/debian-security`.

With the current logic, the generated entries will be
```
deb http://deb.debian.org/debian bookworm main
deb http://deb.debian.org/debian bookworm-updates main
deb http://security.debian.org/debian-security bookworm-security main
```
while the desired ones according to the new docs are
```
deb http://deb.debian.org/debian bookworm main
deb http://deb.debian.org/debian bookworm-updates main
deb http://deb.debian.org/debian-security bookworm-security main
```

Reply:
Just being curious, and according to https://wiki.debian.org/SourcesList, I assume they should be interchangeable (that's why it worked). Nevertheless, I will use what they state in the official release page you mentioned.